apps/ui/src/views/Settings.vue

```diff
@@ -1,5 +1,20 @@
 <script setup lang="ts">
-const { web3Account } = useWeb3();
+const { web3, web3Account } = useWeb3();
+const { modalAccountWithoutDismissOpen } = useModal();
+
+watch(
+  [() => web3.value.account, () => web3.value.authLoading],
+  ([account, authLoading]) => {
+    if (!account && !authLoading) {
+      modalAccountWithoutDismissOpen.value = true;
```

Comment:
Maybe better to redirect to home/explore page?

Reply:
My reasoning is as follow:

user who wants these page have a direct link (bookmarks, etc…), and want their content. If not logged in, they would want to login, else it would not make sense to access these pages.


/home is different, as there’s a link from the landing page, and anyone can go there.

Reply:
@bonustrack wdyt?

Reply:
Let's merge this for now

Reply:
Make sense, I'll merge this